pkg/apis/v1beta1/suite_test.go

```diff
@@ -41,7 +42,7 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
-	env = coretest.NewEnvironment(coretest.WithCRDs(apis.CRDs...))
+	env = coretest.NewEnvironment(coretest.WithCRDs(apis.CRDs...), coretest.WithCRDs(v1alpha1.CRDs...))
```

Comment:
You shouldn't need this. `apis.CRDs` should include the `v1alpha1.CRDs`

Reply:
see: https://github.com/aws/karpenter-provider-aws/blob/main/pkg/apis/apis.go#L34

Reply:
To model the testing behavior really close to how it would happen in reality, I ended up adding a `TestNodeClass` CRD in upstream. I also updated the package name to v1alpha1 for that right here: https://github.com/kubernetes-sigs/karpenter/tree/main/pkg/test/v1alpha1/crds
And because of that I think `apis.CRDs` will not include `v1alpha1.CRDs`.

Reply:
Gotcha makes sense